pkg/security/config/config.go

```diff
@@ -315,6 +315,10 @@ func NewConfig(cfg *config.Config) (*Config, error) {
 		}
 	}
 
+	if c.EventStreamBufferSize%os.Getpagesize() != 0 || c.EventStreamBufferSize&(c.EventStreamBufferSize-1) != 0 {
```

Comment:
nit: I think `c.EventStreamBufferSize == 0` would work here (meaning not trigger an error)

Reply:
Yes, this is on purpose. It will then use the default value